corehq/ex-submodules/couchexport/writers.py

```diff
@@ -405,10 +405,14 @@ def get_write_value(value):
                 value = ''
             return dirty_chars.sub('?', value)
 
-        cells = [WriteOnlyCell(sheet, get_write_value(val)) for val in row]
+        write_values = [get_write_value(val) for val in row]
+        cells = [WriteOnlyCell(sheet, val) for val in write_values]
         if self.format_as_text:
             for cell in cells:
                 cell.number_format = numbers.FORMAT_TEXT
+        for hyperlink_column_index in hyperlink_column_indices:
```

Comment:
Looks like `hyperlink_column_indices` could be `None`, which would cause an error. Could change the default from `None` to `()` (empty tuple).